modulemd/v2/modulemd-module.c

```diff
@@ -389,6 +389,15 @@ modulemd_module_add_stream (ModulemdModule *self,
 }
 
 
+GStrv
+modulemd_module_index_get_stream_names_as_strv (ModulemdModule *self)
+{
+  g_return_val_if_fail (MODULEMD_IS_MODULE (self), NULL);
+
+  return modulemd_ordered_str_keys_as_strv ((GHashTable *)self->streams);
```

Comment:
`self->streams` is a `GPtrArray`, not a `GHashTable`. You can't get at them this way.

The simplest approach would be to create a new `GHashTable` (call it `stream_names`)in this function and iterate through `self->streams`, calling `g_hash_table_add(stream_names, stream_name_from_index)`

Then return `modulemd_ordered_str_keys_as_strv(stream_names)`.